packages/ezxenstore/ezxenstore.0.4.3/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "jonathan.ludlam@citrix.com"
+authors: ["xen-api@lists.xensource.com"]
+license: "ISC"
+homepage: "https://github.com/xapi-project/ezxenstore"
+bug-reports: "https://github.com/xapi-project/ezxenstore/issues"
+dev-repo: "git+https://github.com/xapi-project/ezxenstore.git"
+build: [[ "dune" "build" "-p" name "-j" jobs ]]
+depends: [
+  "ocaml"
+  "dune" {>= "1.4"}
+  "cmdliner" {with-test & >= "1.1.0"}
+  "logs"
+  "uuidm"
+  "xenctrl"
+  "xenstore"
+  "xenstore_transport"
+]
+synopsis:
+  "An easy-to-use interface to xenstore"
+description: """
+An easy-to-use xenstore library with a simplified interface geared
+towards use within a daemon that maintains a single connection to
+xenstored."""
+url {
+  src:
+    "https://github.com/xapi-project/ezxenstore/releases/download/0.4.3/ezxenstore-0.4.3.tbz"
```

Comment:
This url 404s

Reply:
Oops, forgot to run the release step, should be available now